didcomm/pack.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+
+from typing import List, Optional
+
+from didcomm.interfaces.did_resolver import DIDResolver
+from didcomm.interfaces.secrets_resolver import SecretsResolver
+from didcomm.types.algorithms import AnonCryptAlg, AuthCryptAlg, SignAlg
+from didcomm.types.plaintext import Plaintext
+from didcomm.types.types import JSON, DID, DID_OR_KID
+
+
+class Packer:
+    """Packer of plaintext DIDComm messages."""
+
+    def __init__(self,
+                 secrets_resolver: SecretsResolver,
+                 did_resolver: DIDResolver):
+        """Creates a Packer instance."""
+        pass
+
+    async def sign(self,
+                   plaintext: Plaintext,
+                   frm: DID_OR_KID,
+                   sign_alg: SignAlg) -> JSON:
+        """Packs the plaintext DIDComm message to sing(plain) format."""
+        return ""
+
+    async def anon_crypt(self,
+                         plaintext: Plaintext,
+                         to: List[DID],
+                         enc_alg: AnonCryptAlg) -> JSON:
+        """Packs the plaintext DIDComm message to anoncrypt(plain) format."""
+        return ""
+
+    async def auth_crypt(self,
+                         plaintext: Plaintext,
```

Comment:
All the arguments except `plaintext` are optional there. Please look at the original PR